modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/styles/_liferay.scss

```
@@ -93,6 +71,8 @@ body {
 	max-height: 800px;
 	width: 300px;
 
+	overflow-y: hidden;
```

Change alphabetical order and remove unnecessary line

```suggestion
	overflow-y: hidden;
```